tests/console/shar.pm

```diff
@@ -20,10 +20,9 @@ use version_utils;
 sub run {
     my $self = shift;
 
-    if (is_jeos() || is_public_cloud() || is_sle()) {
-        select_console 'root-console';
-        zypper_call('in sharutils');
-    }
+    select_console 'root-console';
+    assert_script_run('rpm -q sharutils || zypper install -y sharutils');
```

Comment:
As a note, we have `_ensure_installed_zypper_fallback` in `lib/susedistribution`, so we could reuse it. Also, `zypper in %pkg%` return `0` in case package is already installed, so it's safe to call it straight away, unless package is not available and might be installed from other sourced.

Reply:
will do a trial with direct zypper_call ...
pending verification